### PR TITLE
Add Trym.app v1.1.0

### PR DESCRIPTION
**Pull request by brandonweiss**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by commitay** [comment]
Where did you get the download link? All I can see on the homepage is a buy now button.

trym

**Comment by brandonweiss** [comment]
The way the author designed the app it immediately asks you to enter your serial number, so there's no point in downloading it until you've bought it. So you get an email with your serial number and a link to download the app. I bought the app so I knew what the link was.

**Comment by commitay** [comment]
Ping @vitorgalvao

File: Casks/trym.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  version '1.1.0'
+  sha256 'e277977e54070760b29de0ccc6ce549d0fe2b36f930c5a2208191f050224d991'
+
+  url "http://kontentapps.com/releases/trym-#{version.dots_to_hyphens}.dmg"
+  name 'Trym'
+  homepage 'http://kontentapps.com/trym'
+
+  app 'Trym.app'
+
+  zap delete: [
```
Comment (commitay) [inline Casks/trym.rb]: Please change this to `zap trash`

**Review by commitay — DISMISSED** [review]

File: Casks/trym.rb (hunk 1 above)
Comment (brandonweiss) [reply Casks/trym.rb]: Will do!

**Comment by brandonweiss** [comment]
Updated!

**Comment by vitorgalvao** [comment]
It’s not a walled build, so it can go in.

**Pull request merged**